lib/src/visitor/serialize.dart

```diff
@@ -802,8 +822,9 @@ class _SerializeCssVisitor
   /// Writes indentation based on [_indentation].
   void _writeIndentation() {
     for (var i = 0; i < _indentation; i++) {
-      _buffer.writeCharCode($space);
-      _buffer.writeCharCode($space);
+      for (var j = 0; j < _indentWidth; j++) {
+        _buffer.writeCharCode(_indentCharacter);
```

Comment:
Yes, I believe it is. https://github.com/sass/node-sass/blob/c048687bf5df536b298154a72939438ea7be0b58/lib/index.js#L110..L137